official/resnet/resnet_run_loop.py

```diff
@@ -636,6 +638,15 @@ def define_resnet_flags(resnet_size_choices=None):
       name='turn_off_distribution_strategy', default=False,
       help=flags_core.help_wrap('Set to True to not use distribution '
                                 'strategies.'))
+  flags.DEFINE_enum(
```

Comment:
Can we please make sure to test all the code paths before merging this change? Namely: all combinations of keras/estimator, cifar/imagenet, all values of old flag/new flag.
It would be much more costly to discover bugs in this in the future, especially given this stuff has hardly any unittest coverage unfortunately.